src/formatly.test.ts

```diff
@@ -77,4 +91,36 @@ describe("formatly", () => {
 			result: { code: 0, signal: null },
 		});
 	});
+
+	it("uses the preferred package manager", async () => {
+		const mockFormatter = formatters[0];
+		mockResolveFormatter.mockResolvedValueOnce(mockFormatter);
+
+		const report = await formatly(patterns);
+
+		expect(report).toEqual({
+			formatter: mockFormatter,
+			ran: true,
+			result: { code: 0, signal: null },
+		});
+		expect(mockSpawn).toHaveBeenCalled();
+		// eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
+		expect(mockSpawn.mock.lastCall?.at(0)).toEqual("npx");
```

Comment:
[Testing] Not a blocker, just a suggestion if you want 🙂 

```suggestion
		expect(mockSpawn).toHaveBeenCalledWith("npx", expect.any(Array), {
			cwd: expect.any(String),
		});
```
```